jvm-libs/linea/blob-compressor/src/test/kotlin/linea/blob/GoBackedBlobCompressorTest.kt

```diff
@@ -41,14 +41,20 @@ class GoBackedBlobCompressorTest {
 
   @Test
   fun `test compression data limit exceeded`() {
-    val blocks = TEST_DATA.iterator()
+    var blocks = TEST_DATA.iterator()
     var result = compressor.appendBlock(blocks.next())
-    while (result.blockAppended && blocks.hasNext()) {
+    // at least one block should be appended
+    assertThat(result.blockAppended).isTrue()
+    while (result.blockAppended) {
       val blockRlp = blocks.next()
       val canAppend = compressor.canAppendBlock(blockRlp)
       result = compressor.appendBlock(blockRlp)
       // assert consistency between canAppendBlock and appendBlock
       assertThat(canAppend).isEqualTo(result.blockAppended)
+      if (!blocks.hasNext()) {
```

Comment:
this is not the ideal, but don't have time to fix all of it now.

Ideally should create static blocks in the test that fill the limit, instead of relying on the ones in the sample.